frontend/src/metabase/dashboard/components/DashboardHeader.tsx

```diff
@@ -90,7 +90,11 @@ const DashboardHeader = ({
         className="Header-buttonSection"
         isNavBarOpen={isNavBarOpen}
       >
-        {headerButtons}
+        {headerButtons.map((headerButton, index) => (
```

Comment:
Could you explain this change?

Reply:
I responded to @uladzimirdev with this, what do you think?

> Ah ignore this. my local tests weren't passing because of the whole each child must have a key when it was just {headerButtons} so I did this as a bandaid that I would take out. Do you think it would make more sense to keep this here and move it to a styled component? Then it would at least get rid of one of the errors